docs/model_developers/developer_getting_started.rst

```diff
@@ -205,12 +205,14 @@ For a v2 module, use the v2 cookiecutter template, then install the module:
 
     pip install cookiecutter
     cookiecutter --checkout v2 https://github.com/inmanta/inmanta-module-template.git
-    inmanta module install -e ./inmanta-module-<module-name>
+    inmanta module install -e ./<module-name>
 
 If you want to use the v2 module in a project, make sure to set up a v2 module source as outlined in the section above,
 then add the module as a dependency of the project as described in :ref:`dgs-existing-module`.
 The location of the module directory is not important for a v2 module.
 
+The Inmanta module with name <module-name> will be distributed as a Python package with name inmanta-module-<module-name>.
+
```

Comment:
I don't think this is really relevant here. Optionally, you could mention right below the code block that this installs a Python package with that name in the active environment. But I would not talk about distribution, that is only relevant in the other section.